src/Sulu/Bundle/AdminBundle/Resources/js/containers/FormOverlay/formOverlay.scss

```diff
@@ -1,7 +1,6 @@
 @import '../Application/variables.scss';
 
 .form {
-    margin: $viewPaddingVertical $viewPaddingHorizontal;
-    overflow: hidden;
-    position: relative;
```

Comment:
The position relative and overflow hidden was introduced in with the snackbar component in https://github.com/sulu/sulu/pull/4717

But in the same pull request the snackbar was moved from content part of the overlay to the main part and so the change itself was not required and so it is obsolete. And as written in the comment the overflow hidden makes problems here in comby with position relative.